tests/acceptance/01_vars/01_basic/os_name_human.cf

```diff
@@ -0,0 +1,54 @@
+body common control
+{
+  bundlesequence => { "test", "check" };
+}
+
+bundle agent test
+{
+  meta:
+      "description" -> { "CFE-3569" }
+        string => "Test for expected values of variable sys.os_name_human";
+
+  # Order is important, the last variable assignment wins.
+  # E.g. prefer Ubuntu over Debian
+  vars:
+    any::
+      "actual" string => "$(sys.os_name_human)";
+    debian::
+      "expected" string => "Debian";
+    ubuntu::
+      "expected" string => "Ubuntu";
+    redhat::
+      "expected" string => "RHEL";
+    centos::
+      "expected" string => "CentOS";
+    fedora::
+      "expected" string => "Fedora";
+    aix::
+      "expected" string => "AIX";
+    hpux::
+      "expected" string => "HPUX";
+    suse::
+      "expected" string => "SUSE";
+    macos::
+      "expected" string => "macOS";
+    windows::
+      "expected" string => "Windows";
+    freebsd::
+      "expected" string => "FreeBSD";
```

Comment:
Should solaris be in here as well ?

Reply:
And maybe NetBSD, OpenBSD as well ?

Reply:
@larsewi Yes, solaris should be added. We don't have to add platforms outside of what we officially support and cover in CI / release testing, but I usually include FreeBSD in my PRs, because I have tested on FreeBSD (manually) before and can do it again. Same goes for macOS. I would probably skip OpenBSD and NetBSD, unless you or someone else can do a quick test and confirm that it works.